tests/unit/test_dirs_core.py

```
@@ -0,0 +1,193 @@
+
+    def test_get_dir_size_handles_permission_errors(self, tmp_path: Path) -> None:
+        """Test get_dir_size handles permission errors gracefully."""
+        test_dir = tmp_path / "test_dir"
+        test_dir.mkdir()
+        shutil.copyfile(FIXTURES_DIR / "small_100.txt", test_dir / "file1.txt")
+
+        try:
+            (test_dir / "file1.txt").chmod(0o000)
+            get_dir_size(str(test_dir), verbose=True)
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)